documentation/Sphinx/user_guide/build-guide/build-guide.rst

```diff
@@ -95,6 +95,10 @@ up a terminal wherein the user can change build options interactively.
       - **ON** / OFF
       - Determines whether the libraries are linked statically (``OFF``) or 
         dynamically (``ON``).
+    * - ``-DSTRING_LENGTH``
+      - *INTEGER*
+      - Maximum length of the identifier string.  Default if
+        unspecified is truncate strings at 100 characters.
```

Comment:
More correct to say default is 100 and vernier will error if length is greater than 100